kubernetes/nebula-graphd-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nebula
```

Comment:
Could deployment/service use same label `nebula` to manage their different pods ?